WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/utils/BarChartAccessibilityHelper.kt

```diff
@@ -0,0 +1,82 @@
+package org.wordpress.android.ui.stats.refresh.utils
+
+import android.graphics.Rect
+import android.os.Bundle
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat.AccessibilityActionCompat
+import androidx.customview.widget.ExploreByTouchHelper
+import com.github.mikephil.charting.charts.BarChart
+import com.github.mikephil.charting.data.BarEntry
+
+class BarChartAccessibilityHelper(
+    private val barChart: BarChart,
+    private val contentDescriptions: List<String>,
+    private val hasOverlappingEntries: Boolean
+) : ExploreByTouchHelper(barChart) {
+    interface BarChartAccessibilityEvent {
+        fun onHighlight(index: Int, hasOverlappingEntries: Boolean)
+    }
+
+    private val dataSet = barChart.data.dataSets.first()
+    var accessibilityEvent: BarChartAccessibilityEvent? = null
```

Comment:
Does this field need to be `var` and nullable? Could we move it into the constructor and change it to `val`?

Reply:
Yes, this a good observation. Will do the refactor!

Reply:
resolved in 4c473f4